packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -118,9 +121,16 @@ import { type PropType, computed, ref, onBeforeMount } from 'vue'
 import { KButton } from '@kong/kongponents'
 import { BookIcon, AddIcon } from '@kong/icons'
 import composables from '../../composables'
-import type { EmptyStateFeature } from 'src/types/entity-empty-state'
+import { type EmptyStateFeature, type AppearanceTypes, Appearances } from '../../types/entity-empty-state'
 
 const props = defineProps({
+  appearance: {
```

Comment:
Please update the [docs](https://github.com/Kong/public-ui-components/blob/e65cadf921de203038b12c02eeaedde46985f1c1/packages/entities/entities-shared/docs/entity-empty-state.md) explaining this new prop